src/Styles/style.ts

```diff
@@ -0,0 +1,39 @@
+import styled from "@emotion/styled";
+import { Padding } from "@mui/icons-material";
+import { AppBar, Button, Card } from "@mui/material";
+
+export const ButtonHollow = styled(Button)({
+  border: "2px solid",
+  borderRadius: "30px",
+  borderColor: "#EADDFF",
+  marginRight: "5px",
+  padding: "5px",
+});
+
+export const ButtonFilled = styled(Button)({
+  backgroundColor: "#6750A4",
+  color: "#FFFFFF",
+  borderRadius: "30px",
+  marginRight: "5px",
+  padding: "10px",
+  "&:hover": {
+    backgroundColor: "#FFFFFF",
+    color: "#6750A4",
+    border: "2px solid",
+    borderRadius: "30px",
+    borderColor: "#EADDFF",
+  },
+});
```

Comment:
You still haven't understood what I was trying to say.
Nevermind, I will connect with you on Monday and explain it